benchmark/websockets/parser.benchmark.js

```diff
@@ -0,0 +1,115 @@
+/*!
+ * ws: a node.js websocket client
+ * Copyright(c) 2011 Einar Otto Stangvik <einaros@gmail.com>
+ * MIT Licensed
+ */
+
+/**
+ * Benchmark dependencies.
+ */
+
+var benchmark = require('benchmark')
+  , Receiver = require('../').Receiver
+  , suite = new benchmark.Suite('Receiver');
+require('tinycolor');
+require('./util');
+
+/**
+ * Setup receiver.
+ */
+ 
+suite.on('start', function () {
```

Comment:
arrows, also for other declarations - might as well add modern code to core if it's going to be added.